lib/hooks/useLocalStorage.ts

```diff
@@ -1,33 +0,0 @@
-import { useState } from "react";
```

Comment:
wondering if this might be good to keep for now? I see it was being used with the nfts